src/containers/AnnotationsPane.jsx

```diff
@@ -0,0 +1,372 @@
+/*
+Annotations Pane
+----------------
+
+TODO: Description
+
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Utility } from '../lib/Utility';
+
+import {
+  setRotation, setScaling, setTranslation, resetView,
+  setViewerState, updateViewerSize, updateImageSize,
+  SUBJECTVIEWER_STATE,
+} from '../ducks/subject-viewer';
+
+const INPUT_STATE = {
+  IDLE: 0,
+  ACTIVE: 1,
+}
+
+//Add ?dev=1 to the URL to enable DEV_MODE
+const DEV_MODE = window.location && /(\?|&)dev(=|&|$)/ig.test(window.location.search);
+
+class SubjectViewer extends React.Component {
```

Comment:
Basically, `containers/AnnotationsPane` is a phantom. `components/AnnotationsPane`, which is used in `containers/SubjectViewer` is the real one. I thought I had moved the AP from components to containers, but something must've screwed up.